apps/emqx_bridge/i18n/emqx_bridge_schema.conf

```
@@ -195,7 +195,7 @@ emqx_bridge_schema {
 
     metric_sent_inflight {
```

nit: inflight is sent
```suggestion
    metric_inflight {
```

---

Will fix it here: https://github.com/emqx/emqx/pull/9642